compiler/damlc/daml-compiler/src/DA/Daml/Compiler/DataDependencies.hs

```diff
@@ -432,21 +475,34 @@ mkDataDecl env thisModule occName tyVars cons = do
 -- | Make a binding of the form "x = x". If a qualified name is passed,
```

Comment:
Please update the doc, now that we are generating error calls. :)

Reply:
good idea, updated